src/common/runner/runner.go

```diff
@@ -189,7 +194,7 @@ func (r *Runner) TagFile(file string) {
 			if r.isSkippedResource(block.GetResourceID()) {
 				continue
 			}
-			if block.IsBlockTaggable() {
+			if block.IsBlockTaggable() && parser.Name() != "Dockerfile" {
```

Comment:
@sshintaku I think the best way around this here is to have each Dockerfile as a single block, with `IsBlockTaggable` being `true` always